libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -24,6 +24,9 @@
       "next": "Næste",
       "previous": "Forrige"
     },
+    "fieldValidation": {
+      "required": "Feltet er påkrævet."
```

Comment:
question: `shared.fieldRequiredErrorText` key (line 22) has the exact same value. After the changes here, is that key still used somewhere?